README.md

```diff
@@ -328,18 +328,27 @@ curl -u "username:access_key" -X GET https://api.browserstack.com/5/workers
  
 Example response:
 ```javascript
-[{
- "id": "<workerId>",
```

Comment:
Do we mention where this ID can be found from? If not maybe, we need a table with legends about each of these parameters. We do need a table since what are other values in case of real_mobile or device? When would I put it as NULL?

Reply:
Shweta, this is autogenerated ID that comes with response. But yes, we can add a table stating the response parameters

Reply:
Shweta, I see some of these parameters are explained in the "Create a new browser worker" API req. Do we still need a table?

Reply:
Not required since it is a response. The edits look good.